### ISSUE TITLE
Support interfaces

### ISSUE THREAD

**Issue by rstormsf** [sc-forks/solidity-coverage#162]
if I have an interface instead of `contract` solidity coverage breaks:

```
There was a problem instrumenting ./coverageEnv/contracts/interfaces/IProxyStorage.sol: TypeError: Cannot read property 'name' of undefined
```
if I change the `interface` to `contract` it works fine

**Comment by cgewecke** [comment]
Thanks @rstormsf! Will look into this...

**Comment by cgewecke** [comment]
@rstormsf An interim solution that might work for you: with `0.4.3` you should now be able to skip entire folders using the `skipFiles` option. For example:
```javascript
skipFiles: ['SomeFile.sol', 'interfaces']
```
I believe you could do this safely / not lose meaningful coverage data since the interfaces are  descriptive and not tested as such.

**Comment by cgewecke** [comment]
@rstormsf Apologies for the delay getting this in. Should be available in the next release.

**Comment by cgewecke** [comment]
Published with `0.4.12`
